### PR TITLE
Added Party Token project to homepage

### PR DESCRIPTION
**Pull request by msb111**
Added Party Token project to homepage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,7 +43,6 @@
 <a href="bridge.html">Bridge Spec</a>
 <a href="whalegate.html">Whalegate</a>
 <a href="staycheap.html">Will it stay cheap?</a>
-<a href="l2.html">cheapETH has L2</a>
```
Comment (flyerman) [inline index.html:46]: Why have you removed this link?

Comment (msb111) [reply index.html:46]: hi, geohot did here: https://github.com/cheapETH/cheapeth-website/commit/1861375d18be49f95914d6ec660d9e9dd504b69c 

I made my branch from the master of cheaeth/cheapeth-website, maybe that's why. Donno if it should be on homepage or not.

Comment (msb111) [reply index.html:46]: I can redo the PRs to CheapEthereum:master to get rid of the conflicts if it helps you. Ping me

Comment (msb111) [reply index.html:46]: I just realized something. I'll close this PR and open a new one at cheapetereum.github.io:master. Sorry for the inconvinience, I thought this was just a clone of the old site.

Be back shortly

**Pull request closed without merging**